### PR TITLE
Requirements milestone

### LINKED ISSUE #8: Define Tech Stack
Identify the tech stack you plan to use. If this is the tech stack required by your client, then state that in the document. If your client is flexible, you must indicate a clear rationale of the choice made. This means an extra page with a table of at least 3 options and document all pros and cons for each before letting your client make a choice.

### PR DESCRIPTION
**Pull request by jpabadir**
close #8

### CODE REVIEW

File: documentation/tech-stack.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+| DB        | Firebase                                                                              | - Client first choice - JP has experience - Easy to learn and use - Large community                  | - Expensive when scaling                                                                           |
+| DB        | AWS                                                                                   | - Large suite of tools (authentication, data processing) - More capable & customizable than Firebase | - Steeper learning curve - Harder to set up                                                        |
+| DB        | MongoDB                                                                               | - Harder to set up than AWS                                                                          | - More predictable pricing                                                                         |
```
Comment (KatrinaMart) [inline documentation/tech-stack.md:14]: Currently when I view the file, it looks like the statements for MongoDB's pros and cons might be reversed. I think "more predictable pricing" is a pro, and "harder to set up than AWS" is a con.

File: documentation/tech-stack.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+
+## Final choice
+### The final tech choice will be the following: 
+- Front-end: React + UI libraries
+- Back end: NodeJS
+- Database and authentication: Firebase
+
+### Rationale:  
+Our client told us that the tech stack we use is, eventually, up to us. However, the client did seem to have a preference for the tech stack above. This is a big reason why we opted for this final choice. In addition, the table above shows some of the significant avantages that our chosen tech stack presents.
```
Comment (KatrinaMart) [inline documentation/tech-stack.md:24]: ``` In addition, the table above shows some of the significant avantages that our chosen tech stack presents. ```
The word advantages has a typo

File: documentation/tech-stack.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+### Rationale:  
+Our client told us that the tech stack we use is, eventually, up to us. However, the client did seem to have a preference for the tech stack above. This is a big reason why we opted for this final choice. In addition, the table above shows some of the significant avantages that our chosen tech stack presents. 
+
+In addition to React, we will be using vraious front-end libraries, such as: Ant design, bootstrap, material UI, bulma, or tailwind CSS. Again, this is a suggestion from our client. Using such front-end libraries has a few drawbacks. First, it will add a bit of complexity to our setup, and dependencies on external codebases, which means we are vulnerable to the future potential lack of maintenance or breaking changes of libraries. However, using such libraries will help us craft a professional and good-looking front-end, with code that is less error-prone than ours, since it is "tried-and-tested" by many developers. In addition, we believe that using well-supported libraries with large communities of developers relying on them (such as Bootstrap or Ant design) will mitigate the issue of being dependent on other developers to maintain the libraries.
```
Comment (KatrinaMart) [inline documentation/tech-stack.md:26]:
```
In addition to React, we will be using vraious front-end libraries, 
```
The word "various" has a typo.

**Review by KatrinaMart — CHANGES_REQUESTED** [review]
Looks good overall, just a few minor changes!

File: documentation/tech-stack.md (hunk 1 above)
Comment (wrc9817) [reply documentation/tech-stack.md:14]: Agree

**Review by aleksssk — APPROVED** [review]
Good

**Review by aleksssk — APPROVED** [review]
Good

**Review by aleksssk — APPROVED** [review]

**Pull request merged**